paths/cases.yaml

```diff
@@ -7,7 +7,7 @@ get:
   parameters:
     - $ref: '../parameters/Code.yaml'
     - $ref: '../parameters/filter/Cases.yaml'
-    - $ref: '../parameters/pagination/Limit.yaml'
+    - $ref: '../parameters/pagination/ExtendedLimit.yaml'
```

Comment:
disagree. I suggest two ways on your choice:
- up limit to 100 for all
- extend $ref by merging it with custom schema

Reply:
pr updated, set max 100